polymesh-docs/primitives/nfts.mdx

```diff
@@ -46,4 +46,86 @@ Once the dispatchable succeeds, the non-fungible token will no longer be linked
\ No newline at end of file
+Polymesh's [Settlement](https://developers.polymesh.network/polymesh-docs/primitives/settlement/#overview) engine fully supports NFTs, and transferring of non-fungible tokens go through the same process as fungible assets. This means that all compliance rules defined for the underlying asset must be respected for a successful transfer of an NFT.
+
+## Metadata Schema
+
+NFTs can be associated with metadata at both the collection and individual NFT granularity. Each NFT collection can define local metadata keys to use for their collection.
+
+There are also 4 globally defined metadata keys, which are intended to be used in a standardised way, to allow third-party dApps etc. to easily process metadata associated with NFTs.
+
+Unlike other chains and standards (e.g. EIP-721) Polymesh encourages you to store metadata directly on-chain - this has the benefit of transparency, reducing dependencies on external systems such as IPFS, and allowing the metadata to be referenced by smart contract business logic.
+
+However, in some cases it may still be preferable to store some metadata off-chain - for compatability purposes or otherwise.
+
+The below two standard approaches to specifying a Token URI and an Image URI are optional, but encouraged if you are storing this data off-chain.
+
+### Token URI
+
+The Token URI of an NFT is intended to point towards an off-chain URI containing a JSON blob defining some metadata of the NFT.
+
+The Token URI can be specified in various ways:
+* via the global metadata key `tokenURI` defined for an individual NFT
+* via the global metadata key `baseTokenURI` defined for the whole NFT collection
+
+In both cases, the metadata value can include `{tokenId}` which will be substituted by a client with the corresponding `tokenId` of the individual NFT being referenced.
+
+The full process to determine the Token URI of an individual NFT is:
+* if the individual NFT has a `tokenURI` value set, use that
+* else if the NFT collection has a `baseTokenURI` value set then:
+  * if the `baseTokenURI` does not already include `{tokenId}` append `/{tokenId}` and use that (e.g. the resolved URI is {baseTokenUri}/{tokenId})
+  * otherwise use the unmodified `baseTokenURI`
+* substitute any `{tokenId}` references with the underlying `tokenId` of the individual NFT
+
+The JSON blob referenced by an individual NFTs Token URI should confirm to the JSON schema below:
+```json
+{
+    "title": "Token Metadata",
+    "type": "object",
+    "properties": {
+        "name": {
+            "type": "string",
+            "description": "Identifies the asset which this token represents"
+        },
+        "description": {
+            "type": "string",
+            "description": "Describes the asset which this token represents"
+        },
+        "image": {
```

Comment:
People are free to include "image" if they want, but I think we should officially recommend using the global metadata that is explicitly labelled for images.

I think the example "image" entry should be removed here

Reply:
Resolved here https://github.com/PolymeshAssociation/polymesh-documentation-site/pull/36/files